src/app/portlets/dot-edit-page/content/components/dot-edit-page-workflows-actions/dot-edit-page-workflows-actions.component.ts

```diff
@@ -1,26 +1,31 @@
-import { Component, OnInit, Input } from '@angular/core';
+import { Component, Input, OnChanges, SimpleChanges } from '@angular/core';
+import { Observable } from 'rxjs/Observable';
 import { MenuItem } from 'primeng/primeng';
 import { DotWorkflowAction } from '../../../../../shared/models/dot-workflow-action/dot-workflow-action.model';
 import { DotWorkflowService } from '../../../../../api/services/dot-workflow/dot-workflow.service';
 import { DotHttpErrorManagerService } from '../../../../../api/services/dot-http-error-manager/dot-http-error-manager.service';
-import { Observable } from 'rxjs/Observable';
+import { DotPage } from '../../../shared/models/dot-page.model';
 
 @Component({
     selector: 'dot-edit-page-workflows-actions',
     templateUrl: './dot-edit-page-workflows-actions.component.html',
     styleUrls: ['./dot-edit-page-workflows-actions.component.scss']
 })
-export class DotEditPageWorkflowsActionsComponent implements OnInit {
-    @Input() inode: string;
+export class DotEditPageWorkflowsActionsComponent implements OnChanges {
+    @Input() page: DotPage;
     @Input() label: string;
 
+    inode: string;
```

Comment:
we dont need this attribute we can use directly:  this.page.workingInode

Reply:
I'll prefer to leave it, to keep just inode more readable instead of a large variable name

Reply:
Is up to you if you want to keep it, but if you are not using this in the template you can make it private, BUT for my experience, duplicated data just make things harder to maintain or test.